client/file_logging.py

```diff
@@ -0,0 +1,138 @@
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+try:
+	import logging
+except ImportError:
+	has_logger = False # catch standard ImportError and set has_logger to False
+else:
+	has_logger = True # no errors, so logger has been imported
+
+try:
+	import os
+except ImportError:
+	has_os = False #catch standard ImportError and set has_os to False
+else:
+	has_os = True # no errors, so os has been imported
+
+# default values for configuration options
+DEFAULT_LOG_DIR = os.path.expanduser('~/.config/king-phisher/logs')
+DEFAULT_LOG_FILE_SIZE = 10
+DEFAULT_LOG_FILE_COUNT = 2
+
+# minimum integer option values
+MIN_LOG_FILE_SIZE = 1
+MIN_LOG_FILE_COUNT = 1
+
+# logger name and file name values
+LOGGER_NAME = ''
+LOG_FILE_NAME = 'client_log.log'
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Zach Janice']
+	title = 'Logger'
+	description = """
+	Keep logs of campaign feedback and results. The directory
+	of the logged file(s), the file size (in MB) of the log, and
+	the number of log files kept can be specified. Plugin requires
+	the 'logging' and 'os' packages available for Python.
+	"""
+	req_packages = {
+		'logger': has_logger,
+		'os': has_os
+	}
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+	options = [
+		plugins.ClientOptionString(
```

Comment:
I don't think it's necessary for these options to be configurable. I'd prefer to see all options removed because they can't be honored until the plugin is disabled and then reenabled either manually or by a restart of the application.